### DISCUSSION TITLE
Playing two videos simultaneously using spacial mapping

### DISCUSSION THREAD

**Discussion by badgrandpa** [mozilla/hubs#4016] [Spoke Questions] [answered]
Hi everyone!

I am new to Spoke and Mozilla hubs and have a quick question. I want two videos to start playing simultaneously when an avatar stands in a certain part of the room. I have read a few forums suggesting using spatial mapping or a box collider, but there is little documentation on how to piece this together. 

Would be great to get some tips or links to further documentation on this.

Thanks a lot!

**Comment by johnshaughnessy** [comment, marked as answer]
Hi @badgrandpa . Right now, we do not have a system that allows you to trigger these kinds of actions when a user enters a collision volume. If you wanted to do something like this, you would need to build it yourself. It has been suggested a number of times to us though, so it's possible we build something to support interactions like these sometime in the future. This issue adds some requirements for what this feature should look like -- if there's anything you'd like to add, please feel do https://github.com/mozilla/hubs/issues/3044
